packages/backend/src/tokens/__tests__/request.test.ts

```diff
@@ -972,130 +972,138 @@ describe('tokens.authenticateRequest(options)', () => {
     expect(requestState.toAuth()).toBeSignedInToAuth();
   });
 
-  test('refreshToken: returns signed in with valid refresh token cookie if token is expired and refresh token exists', async () => {
-    server.use(
-      http.get('https://api.clerk.test/v1/jwks', () => {
-        return HttpResponse.json(mockJwks);
-      }),
-    );
-
-    // return cookies from endpoint
-    const refreshSession = vi.fn(() => ({
-      object: 'token',
-      jwt: mockJwt,
-    }));
-
-    const requestState = await authenticateRequest(
-      mockRequestWithCookies(
-        {
-          ...defaultHeaders,
-          origin: 'https://example.com',
-        },
-        { __client_uat: `12345`, __session: mockExpiredJwt, __refresh_MqCvchyS: 'can_be_anything' },
-      ),
-      mockOptions({
-        secretKey: 'test_deadbeef',
-        publishableKey: PK_LIVE,
-        apiClient: { sessions: { refreshSession } },
-      }),
-    );
-
-    expect(requestState).toBeSignedIn();
-    expect(requestState.toAuth()).toBeSignedInToAuth();
-    expect(refreshSession).toHaveBeenCalled();
-  });
-
-  test('refreshToken: does not try to refresh if refresh token does not exist', async () => {
-    server.use(
-      http.get('https://api.clerk.test/v1/jwks', () => {
-        return HttpResponse.json(mockJwks);
-      }),
-    );
-
-    // return cookies from endpoint
-    const refreshSession = vi.fn(() => ({
-      object: 'token',
-      jwt: mockJwt,
-    }));
-
-    await authenticateRequest(
-      mockRequestWithCookies(
-        {
-          ...defaultHeaders,
-          origin: 'https://example.com',
-        },
-        { __client_uat: `12345`, __session: mockExpiredJwt },
-      ),
-      mockOptions({
-        secretKey: 'test_deadbeef',
-        publishableKey: PK_LIVE,
-        apiClient: { sessions: { refreshSession } },
-      }),
-    );
-    expect(refreshSession).not.toHaveBeenCalled();
-  });
-
-  test('refreshToken: does not try to refresh if refresh exists but token is not expired', async () => {
-    // return cookies from endpoint
-    const refreshSession = vi.fn(() => ({
-      object: 'token',
-      jwt: mockJwt,
-    }));
-
-    await authenticateRequest(
-      mockRequestWithCookies(
-        {
-          ...defaultHeaders,
-          origin: 'https://example.com',
-        },
-        // client_uat is missing, need to handshake not to refresh
-        { __session: mockJwt, __refresh_MqCvchyS: 'can_be_anything' },
-      ),
-      mockOptions({
-        secretKey: 'test_deadbeef',
-        publishableKey: PK_LIVE,
-        apiClient: { sessions: { refreshSession } },
-      }),
-    );
-
-    expect(refreshSession).not.toHaveBeenCalled();
-  });
-
-  test('refreshToken: uses suffixed refresh cookie even if un-suffixed is present', async () => {
-    server.use(
-      http.get('https://api.clerk.test/v1/jwks', () => {
-        return HttpResponse.json(mockJwks);
-      }),
-    );
+  describe('refreshToken', async () => {
```

Comment:
I moved these tests into a separate describe block, consider looking at the diff without whitespace to see the actual changes: https://github.com/clerk/javascript/pull/4884/files?w=1